apps/web/composables/useMakeOrder/useMakeOrder.ts

```diff
@@ -47,6 +48,7 @@ export const useMakeOrder: UseMakeOrderReturn = () => {
       await useSdk().plentysystems.doAdditionalInformation(additionalParams);
     } catch (error) {
       handleMakeOrderError(error);
+      return {} as Order;
```

Comment:
might want to move that into the handleMakeOrderError and return that here. Also, it should return null instead of Order

Reply:
yeah but it would at least be consistent and worth it imo